test/test_processing_utils.py

```diff
@@ -194,6 +194,30 @@ def test_make_temp_copy_if_needed(self, mock_copy):
         )
         assert len(temp_file_manager.temp_files) == 2
 
+    def test_base64_to_temp_file_if_needed(self):
```

Comment:
I would test the case of the same audio file but with different min/max values. That's an interesting case I would not have thought of!

Reply:
Good idea thanks!

Reply:
Added a test in `test/test_components`